tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/TemplateEngineTest.cs

```diff
@@ -674,6 +674,41 @@ public void TestEvalExpression()
             Assert.AreEqual(evaled, "goodmorning");
         }
 
+        [TestMethod]
+        public void TestLGResource()
+        {
+            var lgResource = LGParser.Parse(File.ReadAllText(GetExampleFilePath("2.lg")));
```

Comment:
In your test case, we should test, if i add a template with body = "some string", then from templates i can get the same body string there.